bigbluebutton-html5/imports/ui/components/actions-bar/styles.scss

```diff
@@ -30,3 +30,7 @@
 .invisible {
   visibility: hidden;
 }
+
+.isTalking {
+  text-shadow: 0 0 .35rem #FFF;
```

Comment:
Traditionally it's always been the icon that had the glow effect. We left the label alone in the past.